<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientFileWidget</class>
 <widget class="QWidget" name="ClientFileWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btnConnect">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>80</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>连接</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>服务器Port</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPort">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>50</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>777</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditIP">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>90</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>127.0.0.1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>服务器IP</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>150</y>
     <width>281</width>
     <height>21</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
